moderators.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Moderators"
+---
+
+A [moderator](governance.md#moderator) is a volunteer gatekeeper who manages contributions and discussions from [contributors](governance.md#contributor) and [users](governance.md#user).
+[Moderators](governance.md#moderator) are vital to the health and success of R-multiverse: they help the R community understand how to use R-multiverse, they ensure new [package contributions](contributors.md) are reviewed in a timely manner, they maintain the integrity of the R-multiverse package ecosystem.
+If you would like to help improve R-multiverse, please consider becoming a [moderator](governance.md#moderator).
+This page explains the how to become a [moderator](governance.md#moderator) and the scope of the [moderator](governance.md#moderator) role.
+
+# Joining
+
+This section describes the process to become a [moderator](governance.md#moderator).
+
+## Prerequisites
+
+[Moderators](governance.md#moderator) must be:
+
+1. Proficient in [R package development](https://r-pkgs.org/).
+1. Proficient in the English language.
+1. Able to manage content on [GitHub](https://github.com) such as repositories, pull requests, issues, and discussions.
```

Comment:
How about linking to https://happygitwithr.com/ instead?

Reply:
done: 3544419e835370dd0a553d0ca48fcdf4e0083016